cellfinder/core/train/train_yml.py

```diff
@@ -338,7 +338,7 @@ def run(
 
     ensure_directory_exists(output_dir)
     model_weights = prep_model_weights(
```

Comment:
well spotted!!
should we explicitly add the keywords, to try to prevent this happening in the future?

```
model_weights = prep_model_weights(
  model_weights=model_weights,
  install_path=install_path,
  model_name=model,
  n_free_cpus=n_free_cpus,
 )
```
(sorry just pasting it cause it won't allow code suggestions on "deleted" lines)

Reply:
Good idea!